tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -773,6 +773,144 @@ where
         }
         Ok(())
     }
+
+    /// Returns metadata for the span with the given `id`, if it exists.
+    ///
+    /// If this returns `None`, then no span exists for that ID (either it has
+    /// closed or the ID is invalid).
+    ///
+    /// <div class="information">
+    ///     <div class="tooltip ignore" style="">ⓘ<span class="tooltiptext">Note</span></div>
+    /// </div>
+    /// <div class="example-wrap" style="display:inline-block">
+    /// <pre class="ignore" style="white-space:normal;font:inherit;">
+    /// <strong>Note</strong>: This requires the wrapped subscriber to implement the
+    /// <a href="../registry/trait.LookupSpan.html"><code>LookupSpan</code></a> trait.
+    /// <code>Layer</code> implementations that wish to use this
+    /// function can bound their <code>Subscriber</code> type parameter with:
+    /// </pre></div>
```

Comment:
I worry that this informational warning, while useful, is [getting lost](https://deploy-preview-834--tracing-rs.netlify.app/tracing_subscriber/fmt/struct.fmtcontext). Maybe move this information to the [`FmtContext` declaration](https://deploy-preview-834--tracing-rs.netlify.app/src/tracing_subscriber/fmt/fmt_layer.rs.html#730?)?

Reply:
I think the note should actually be removed in this case --- the `fmt` layer already requires `LookupSpan` (and `FmtContext` inherits that requirement) so these methods will always be present.